provisioning/roles/ruby/tasks/main.yml

```diff
@@ -1,7 +1,57 @@
-- name: ensure Ruby packages are installed
-  apt: pkg={{ item }} state=installed
-  become: yes
-  with_items:
-    - ruby
-    - ruby-dev
-    - rubygems-integration
+---
+- name: install rbenv
+  git:
+    repo: https://github.com/rbenv/rbenv.git
+    dest: ~/.rbenv
+    version: v1.1.0
+
+- name: try to compile dynamic bash extension to speed up rbenv
+  command: src/configure && make -C src
+  ignore_errors: yes
+  args:
+    chdir: ~/.rbenv/
+    creates: ~/.rbenv/bin/rbenv
```

Comment:
This will always be true, please check the existence of `~/.rbenv/libexec/rbenv-realpath.dylib` that will be created by the `make` command.